docs/api_reference/contacts_api/get_contacts.md

```diff
@@ -15,11 +15,10 @@ List all contacts belonging to the account. A filter can be specified in order t
 | `source`  | Source   | The integration type (e.g. `whatsapp`)                                             |
 | `tags`    | string[] | The matching tags, comma-separated (e.g. `sales,lead`). Tags are _case-insentive_. |
 
-
 ### Example Request
 
 ```bash title=request.sh
-curl -X GET "https://api.callbell.eu/v1/contacts?page=1&source=whatsapp&tags=sales,lead" \
+curl -X GET "https://api.callbell.eu/v1/contacts" \
```

Comment:
Why did you remove the extra params here? 🤔

Reply:
not sure actually, probably 🌮 🙏🏼